tests/test_link_io_structure.py

```diff
@@ -95,7 +95,30 @@ def test_overwrite_symlink_stdout(tmpdir, capsys):
         os.symlink("2023.06", "2023")
         create_symlinks(links_dict)
         captured = capsys.readouterr()
-        assert "Existing symlink 2023 moved from 2023.06 to 2023.07\n" == captured.out
+        assert (
+            "Existing symlink 2023 moved from 2023.06 to 2023.07"
+            in captured.out.split("\n")
```

Comment:
This assert now leaves the possibility for garbage on subsequent lines. Can you constrain it to not allow that?